README.md

```diff
@@ -42,6 +42,14 @@ Apps you would like to have installed via `cask`. Search for popular apps on htt
 
 Directory where applications installed via `cask` should be installed.
 
+    homebrew_use_brewfile: true
+
+Whether to install via a Brewfile. If so, you will need to install the `homebrew/bundle` tap, which could be done within `homebrew_taps`. 
+
+    homebrew_brewfile_path: '~'
```

Comment:
Can you change this variable to `homebrew_brewfile_dir`, just to be a little more semantically correct? When I see 'path' I think of the full path to the file, not the directory containing the file.